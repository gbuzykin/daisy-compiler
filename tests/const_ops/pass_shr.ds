#if (2678i32 >> 5) != 83i32
#error invalid result
#endif
#if (2678u32 >> 5) != 83u32
#error invalid result
#endif
#if (-2678i32 >> 5) != -84i32
#error invalid result
#endif
#if (4294964618u32 >> 5) != 134217644u32
#error invalid result
#endif
