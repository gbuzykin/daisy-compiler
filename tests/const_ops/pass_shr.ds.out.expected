./const_ops/pass_shr.ds: info: warnings 0, errors 0
